lib/grammar_sandbox.c

```diff
@@ -498,6 +498,8 @@ struct message tokennames[] = {item(WORD_TKN),	// words
 			       item(IPV4_PREFIX_TKN), // IPV4 network prefixes
 			       item(IPV6_TKN),	// IPV6 prefixes
 			       item(IPV6_PREFIX_TKN), // IPV6 network prefixes
+			       item(MAC_TKN),	 // MAC address
+			       item(MAC_PREFIX_TKN),  // MAC address w/ mask
 
```

Comment:
`lib/command_py.c` needs to be updated too, and it'd be nice if you could also do `python/clidef.py`